database/sql/V037__add_onchain_order_cancellation_status.sql

```diff
@@ -0,0 +1,14 @@
+-- Table to store all canellations of onchain orders
+-- uid is needed as unique identifier for the order.
+-- is_reorged is a flag indicated whether the cancellation was mined and then reorged.
+-- block_number + log_index are used to deal with chain reverts.
+
+CREATE TABLE onchain_order_cancellations (
+    uid bytea PRIMARY KEY,
+    is_reorged bool NOT NULL,
```

Comment:
Instead of storing if it is reorged, should we just remove the event?

Reply:
Or better yet, is the `is_reorged = false` => `is_reorged = true` the only direction it can go?

Reply:
Since I can not think of a use case to report reorged invalidations, I removed reorged invalidations completely.